aidants_connect_web/models.py

```diff
@@ -137,6 +139,11 @@ class Usager(models.Model):
     def __str__(self):
         return f"{self.given_name} {self.family_name}"
 
+    def save(self, *args, **kwargs):
+        if self._state.adding:
+            self.sub_fi = token_hex(16)
```

Comment:
Je trouve cela étrange de générer une chaine au hasard VS un hash des informations contenues dans l'instance du modèle. Je me demande pourquoi FC a choisi la seconde option.

Reply:
j'ai trouvé la doc concernant le sub:
- oidc: REQUIRED. Subject Identifier. Locally unique and never reassigned identifier within the Issuer for the End-User, which is intended to be consumed by the Client, e.g., 24400320 or AItOawmwtWwcT0k51BayewNvutrJUqsvl6qs7A4. It MUST NOT exceed 255 ASCII characters in length. The sub value is a case-sensitive string.
- franceconnect: Identifiant technique (unique et stable dans le temps pour un individu donné) fourni par FranceConnect au FS. Le sub est présent dans l'IdToken retourné au FS ainsi que dans les informations d'identité. Le sub retourné par FranceConnect est spécifique à chaque fournisseur de service (i.e: Un usager aura toujours le même sub pour un FS donné, en revanche il aura un sub différent par FS qu'il utilise).

ca semble etre bien aléatoire comme champ